pyproject.toml

```diff
@@ -0,0 +1,7 @@
+[build-system]
+requires = [
+    "setuptools>=42",
+    "wheel",
+    "oteapi-core==0.0.4"
```

Comment:
```suggestion
```
The build system does not have this requirement.
But it should be a requirement/dependency of the package.